addons/binding/org.openhab.binding.heos/README.md

```diff
@@ -0,0 +1,420 @@
+			sendCommand(HeosBridge_Play_Living, OFF)	//Switch player channel off again to be sure that it is OFF
+		}
+```
+
+Sitemap:
+
+```
+Switch item=HeosKitchen_InputSelect	mappings=[aux_in_1 = "Aux In" , LivingRoom = "Living Room"]
+```
+
+### The OnlineState of Groups and Players
+
+The online state of a Thing can be helpful for groups to control the visibility of group items within sitemap. So if the group is removed the visibility of those items is also changed.
+
+#### Example
+
+First you have to define a new Item within the Item section which is used later within the Sitemap:
+
+Items:
+
+```
+String HeosGroup_Status
+
+```
+
+Then we need a rule which triggers the state if an Item goes Online or Offline.
+
+Rules:
+
+```
+rule "Online State Heos Group" 
+
+when
+    Thing "heos:group:1747557118" changed 
+then
+    var thingStatus = getThingStatusInfo("heos:group:1747557118")
+    sendCommand(HeosGroup_Status, thingStatus.getStatus.toString)    
+end
+
+```
+
+Sitemap:
+
+```
+Frame label="Heos Group" visibility=[HeosGroup_Status==ONLINE] {
+	
+	Default item=HeosGroup1_Player
+	Default item=HeosGroup1_Volume
+	Default item=HeosGroup1_Mute
+	Default item=HeosGroup1_Favorites
+	Default item=HeosGroup1_Playlist
+	
+	Text item=HeosGroup1_Song {
+		Default item=HeosGroup1_Song
+		Default item=HeosGroup1_Artist
+		Default item=HeosGroup1_Album
+		Image item=HeosGroup1_Cover url=""
+	}
+	
+}
+```
+
+### Dynamic Group Handling (Keep Groups)
+
+Groups are a more or less dynamic item compared to the players. The players are always present within the network and are only added or removed if they are added or removed physically. Groups on the other hand are dynamic because they are only virtual created and removed. 
+Therefore dynamic group handling only sets groups to OFFLINE if the option is activated. This is indicated via the "OnlineStatus" channel of the group. If the group then is created again the binding automatically sets the status to online without the need of action by the user.
+If dynamic group handling is deactivated the group thing is completely removed if the binding is notified about a removed group. The group then has to be added manually again if created.
+
+
+### Playlists
+
+Playlists can be played by sending the number (starts at 0!) to the binding via the playlists channel at the bridge. To find the correct number for the playlist, please have a look to the HEOS App and see at which position the playlist you want to play is located.
+To tell the binding at which player or group the playlist shall be played, select the related player or group channel at the bridge prior sending the command to the playlist channel.
+This can be done by a rule within openHAB
+
+#### Example
+
+Items:
+
+```
+Switch HeosBridge_Play_Kitchen		"Kitchen"   (gHeos) {channel="heos:bridge:ed0ac1ff-0193-65c6-c1b8-506137456a50:P918797451"}
+String HeosBridge_Playlists		"Playlists" (gHeos) {channel="heos:bridge:ed0ac1ff-0193-65c6-c1b8-506137456a50:Playlists"}
```

Comment:
Do you set the playlist to be played on the bridge and not the actual player?

Reply:
Yes. The reason was to have a channel for each favorite. To reduce the amount of channels per thing I decided to have them at the bridge and control playing the favorite from the bridge by selecting the player channel first and then the favorite channel.

Reply:
Now I also understand how I should use the favorites but because of that you cannot control them without any rules (or have to use multiple buttons in the GUI).

What is the consequence of multiple heos devices are one will it be played on all then? I think it would be way easier to use (most likely not for your implementation ;-) ) when you just have those channels on the player and group, so you directly control it without any rules. wdyt?

Reply:
I agree. Now I also think that this is a better solutin. I can see how this can be done. I think it should possible to use the same method which creates the channel on the bridge for creating them on the players and groups. 
And yes. If you select multiple player it starts playing on all of them.

Reply:
Oh big topic... 😄 Yes handling the queue is a topic which could be added. But I haven't started yet. But yes, cleaning the queue should be added. This can be done by a single channel. 
If thinking about further queue handling also a "Queue Channel" with additional options could be a way...

Reply:
Maybe we should park this till after a merge, it is a new feature we could be based on the merged version.

Reply:
One question. Is it possible to add options to a channel which is added dynamically during execution? This would maybe be a cool thing if there is a playlist channel which as options depending on the found playlists...

Reply:
I'm not that familiar to that function maybe @ppieczul or @tmrobert8 or @htreu know :-)